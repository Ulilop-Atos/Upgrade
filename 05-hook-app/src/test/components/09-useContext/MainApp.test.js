import { shallow } from 'enzyme';
import MainApp from '../../../components/09-useContext/MainApp';

describe('Tests for <MainApp />', () => { 
  
  test('should display properly', () => { 
    const wrapper =  shallow(<MainApp />);
    expect(wrapper).toMatchSnapshot();
   });

 });