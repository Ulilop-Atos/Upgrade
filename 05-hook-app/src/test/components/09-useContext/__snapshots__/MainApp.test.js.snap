// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for <MainApp /> should display properly 1`] = `
<ContextProvider
  value={
    Object {
      "setUser": [Function],
      "user": Object {},
    }
  }
>
  <AppRouter />
</ContextProvider>
`;
